odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/config/OAuthSecurityConfiguration.java

```diff
@@ -69,6 +70,7 @@
 @EnableReactiveMethodSecurity
 @EnableWebFluxSecurity
 @RequiredArgsConstructor
+@Slf4j
```

Comment:
seems unnecessary here, could you please revert that?

Reply:
removed